src/pretalx/orga/permissions.py

```diff
@@ -0,0 +1,31 @@
+import rules
+
+from pretalx.person.permissions import is_orga, is_reviewer
+
+rules.add_perm('orga.view_orga_area', is_orga | is_reviewer)
+rules.add_perm('orga.search_all_users', is_orga)
+rules.add_perm('orga.change_settings', is_orga)
+rules.add_perm('orga.view_submission_cards', is_orga)
+rules.add_perm('orga.edit_cfp', is_orga)
+rules.add_perm('orga.view_question', is_orga)
+rules.add_perm('orga.edit_question', is_orga)
+rules.add_perm('orga.remove_question', is_orga)
+rules.add_perm('orga.view_submission_type', is_orga)
+rules.add_perm('orga.edit_submission_type', is_orga)
+rules.add_perm('orga.remove_submission_type', is_orga)
+rules.add_perm('orga.view_mails', is_orga)
+rules.add_perm('orga.send_mails', is_orga)
+rules.add_perm('orga.purge_mails', is_orga)
+rules.add_perm('orga.view_review_dashboard', is_orga | is_reviewer)
+rules.add_perm('orga.view_reviews', is_reviewer)
+rules.add_perm('orga.remove_review', is_reviewer)
+rules.add_perm('orga.view_schedule', is_orga)
+rules.add_perm('orga.release_schedule', is_orga)
+rules.add_perm('orga.edit_schedule', is_orga)
+rules.add_perm('orga.schedule_talk', is_orga)
+rules.add_perm('orga.view_room', is_orga)
+rules.add_perm('orga.edit_room', is_orga)
```

Comment:
This is never actually used. Instead, `RoomDetail`, which is currently part of the general event settings tabs, relies on `orga.change_settings`. But since `orga.view_room` is used, `.edit_room` should be, too.

Reply:
Good point, fixed.